package.json

```diff
@@ -18,11 +18,13 @@
     "test:jest": "jest --watchAll=false"
   },
   "resolutions": {
+    "@testing-library/dom": "10.1.0",
```

Comment:
Incorrect resolution of `@testing-library/dom` results in a lot of those extraneous `act()` warnings. Previously, additional versions were being installed by Storybook and LeafyGreen. Idea from [here](https://stackoverflow.com/a/76089260).